test/index.test.ts

```diff
@@ -39,7 +39,13 @@ describe("Recover the address function of a transaction", () => {
     signature,
     isMessagePrefixed
   );
-  it("should recover the signing address of a transaction from the private key", () => {
+  it("should recover the signing address of a transaction", () => {
     expect(recoveredAddress).toBe(signingAddress);
   });
+  it("should be hexadecimal", () => {
```

Comment:
I think it might be cleaner here to put everything inside the `should recover the signing address of a transaction` test. So all of the set up from line 31- 41 and the following two `expect` statements